src/client/components/input/drag-and-drop-image.js

```diff
@@ -1,11 +1,26 @@
 const React = require('react');
 const Input = require('react-bootstrap').Input;
 
+/**
+* This class is derived from the React Component base class to render an image
+* which supports drag and drop functionality.
+*/
 class DragAndDropImage extends React.Component {
+	/**
+	* Binds the class methods to their respective data.
+	* @constructor
+	*/
 	constructor() {
 		super();
 		this.handleDragStart = this.handleDragStart.bind(this);
 	}
+	/**
+	* Transfers the data of the achievement badge component properties to the
```

Comment:
Rather than rephrasing the whole comment I added a little more context as to where the data ends up, though I'm not sure that it is the change which you intended.